starter-bots/purely-functional-scala/build.sbt

```diff
@@ -0,0 +1,19 @@
+libraryDependencies += "org.scalaz" %% "scalaz-zio" % "1.0-RC4"
+libraryDependencies += "org.typelevel" %% "cats-core" % "1.6.0"
+val circeVersion = "0.10.0"
+
```

Comment:
Could you add the Scala version you are planning to run on?

Reply:
Thanks.  Done in the next push.